### PR TITLE
Accessibility Related New changes

### PR DESCRIPTION
**Pull request by prashelke**
Thank you for your contribution.
Before submitting this PR, please make sure:

- [x] PR description and commit message should describe the changes done in this PR
- [x] Verify the PR is pointing to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [x] Latest Code from master or specific release branch is merged to your branch
- [x] No unwanted\commented\junk code is included
- [x] No new warning upon build solution
- [ ] Code Summary\Comments are added to my code which explains what my code is doing
- [ ] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [ ] Sanity Tests are successfully executed for New and Existing Functionality
- [x] Verify that changes are compatible with all relevant browsers and platforms.
- [x] After creating pull request there should not be any conflicts
- [ ] Resolve all Codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments
- [ ] Update the Help Library document to match any feature changes

## Summary by CodeRabbit

- **New Features**
	- Enhanced the Accessibility Testing feature with options to analyze by Standard or Severity.
- **Refactor**
	- Improved logic for setting visibility and content in Accessibility Testing options.
	- Refined Accessibility Testing analysis logic for better accuracy.
- **Style**
	- Updated labels in the Accessibility Testing section for clarity.

### CODE REVIEW

File: Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs [hunk 1]
```diff
@@ -1631,24 +1633,25 @@ private AxeBuilder CreateAxeBuilder(ActAccessibilityTesting act)
...
-                    string[] ExcludeRules = act.ExcludeRuleList.Where(i => i.Active).Select(i => i.RuleID.ToString()).ToArray();
                     axeBuilder.DisableRules(ExcludeRules);
                 }
             }
-            else
+            else if (act.GetInputParamValue(ActAccessibilityTesting.Fields.Analyzer) == ActAccessibilityTesting.eAnalyzer.BySeverity.ToString())
             {
-                if (act.IncludeRuleList != null && act.IncludeRuleList.Any(i => i.Active))
+                string[] IncludeRules = act.RulesItemsdata.Where(x => sevritylist.Contains(x.Impact.ToLower())).Select(i => i.RuleID.ToString()).ToArray();
+                if (IncludeRules != null && IncludeRules.Any())
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs]: Handled

File: Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs [hunk 2]
```diff
@@ -1658,23 +1661,39 @@
...
             {
-                act.Status = eRunStatus.Failed;
+                if(ActionResult)
+                {
+                    act.Status = eRunStatus.Failed;
+                }
+                else
+                {
+                    act.Status = eRunStatus.Passed;
+                }
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs]: Handled

File: Ginger/Ginger/Actions/ActionEditPages/ActAccessibilityTestingEditPage.xaml [hunk 3]
```diff
@@ -48,29 +48,20 @@
...
+                <StackPanel x:Name="xAnalyzerRadioButtonPnl" Orientation="Horizontal" HorizontalAlignment="Left" VerticalAlignment="Center" Width="300" Margin="18,0,0,0">
                     <UserControlsLib:UCRadioButtons x:Name="xAnalyzerRadioButton" VerticalAlignment="Center" HorizontalAlignment="Left"/>
                 </StackPanel>
             </StackPanel>
 
             <StackPanel x:Name="xStdStack" Orientation="Horizontal" DockPanel.Dock="Top" Margin="0,10,0,0">
-                <Label x:Name="xStdlbl" Grid.Column="0" Style="{StaticResource $LabelStyle}" Content="Standard :"/>
-                <UserControlsLib:MultiSelectComboBox Grid.Column="1" x:Name="xStdCB" Margin="75,0,0,0" Width="300"/>
+                <Label x:Name="xStdlbl" Grid.Column="0" Style="{StaticResource $LabelStyle}" Content="Standard's :"/>
+                <UserControlsLib:MultiSelectComboBox Grid.Column="1" x:Name="xStdCB" Margin="70,0,0,0" Width="300"/>
```
[the remark this answers is not in the document]
Comment (prashelke) [reply Ginger/Ginger/Actions/ActionEditPages/ActAccessibilityTestingEditPage.xaml:59]: done

File: Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs [hunk 4]
```diff
@@ -1595,110 +1597,8 @@ private void ActAccessibility(ActAccessibilityTesting act)
...
-                        act.AddOrUpdateReturnParamActualWithPath(ParamName: "ViolationHelp", ActualValue: violation.Help, violatedNodeIndex.ToString());
-                        act.AddOrUpdateReturnParamActualWithPath(ParamName: "ViolationImpact", ActualValue: violation.Impact, violatedNodeIndex.ToString());
-                    }
-                }
-            }
-            else
-            {
-                act.Status = eRunStatus.Passed;
-            }
+            act.AnalyserAccessibility(Driver, e);
```
Comment (Maheshkale447) [inline Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs:1601]: Please fix spelling

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs (hunk 4 above)
Comment (prashelke) [reply Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs]: done

**Pull request closed without merging**